test/raven.parsers.js

```diff
@@ -601,19 +601,17 @@ describe('raven.parsers', function () {
 
     it('should parse an error with additional information', function (done) {
       try {
-        assert.strictEqual(1, 2);
+        var err = new Error('boom');
+        err.prop = 'value';
+        throw err;
       } catch (e) {
         raven.parsers.parseError(e, {}, function (parsed) {
           parsed.should.have.property('exception');
           parsed.exception[0].stacktrace.should.have.property('frames');
           parsed.should.have.property('extra');
-          parsed.extra.should.have.property('AssertionError');
```

Comment:
Should we not be concerned that someone using this assertion library wouldn't work with Raven in Node 8?